src/System.Private.ServiceModel/tests/Stress/SharedPoolsOfWCFObjects/TestHelpers.cs

```diff
@@ -41,6 +50,37 @@ public static EndpointAddress CreateHttpEndpointAddress()
             return new EndpointAddress(s_httpUrl);
         }
 
+        public static EndpointAddress CreateEndPointDuplexAddress()
+        {
+            return UseHttp ? CreateHttpEndpointDuplexAddress() : CreateNetTcpEndpointDuplexAddress();
```

Comment:
For the future -- I think we may want stress tests to integrate with the Bridge, allowing EndpointAddresses to be generated on the fly to hosts created on demand.  Let's discuss it later when you return.